src/itemviewformaction.cpp

```diff
@@ -55,6 +56,22 @@ void ItemViewFormAction::init()
 	set_keymap_hints();
 }
 
+void ItemViewFormAction::update_head(const std::shared_ptr<RssItem>& item)
+{
+	const auto feedtitle = item_renderer::get_feedtitle(item);
```

Comment:
This is a `std::string`, I wouldn't use `auto` here. Maybe you meant to do a _auto to stick_?

Reply:
No, it's just laziness I guess. I'll favour the reader and specify the type explicitly.